app/views/layouts/mailer.html.erb

```diff
@@ -20,10 +20,10 @@
   </head>
 
   <body>
-    <div><%= image_tag('email/sul_logo.png', height: 25, class: 'sul-logo', alt: 'Stanford Libraries', style: 'margin-left: 18px;') %></div>
+    <div><%= image_tag('/images/sul_logo.png', height: 25, class: 'sul-logo', alt: 'Stanford Libraries', style: 'margin-left: 18px;') %></div>
     <main>
       <div class="sdr-logo">
-        <%= image_tag('email/rosette_sdr.png', height: 28, alt: 'Stanford Digital Repository') %>
+        <%= image_tag('/images/rosette_sdr.png', height: 28, alt: 'Stanford Digital Repository') %>
```

Comment:
leading slash tells rails to pull from public folder (i nested the images into an "images" folder there for organization purposes)